packages/fhir-converter/src/templates/cda/Sections/History_of_Present_Illness.hbs

```diff
@@ -34,7 +34,7 @@
 
                 {{!-- TODO need to add references, author, etc --}}
                 {{#if @encompassingEncounterId}} 
-                    {{>References/DiagnosticReport/encounter.hbs ID=(generateUUID (toJsonString note.act)) REF=(concat 'Encounter/' @encompassingEncounterId)}}
+                    {{>References/DiagnosticReport/encounter.hbs ID=(generateUUID (toJsonString note.act)) REF=(concat 'Encounter/' @encompassingEncounterId.id)}}
```

Comment:
Id.id is redundant?

Reply:
nope. The whole `encompassingEncounterId` object has this structure: 
{ id: string, externalId: object}

Reply:
Right but I am saying that it doesn't follow this in the style guide: https://www.notion.so/metriport/Development-Best-Practices-bb3574d3c83d46dc9914a9e5c5542ca7?pvs=4#1a649268035a4b8ab0457c16553fae9e

Reply:
i'd argue this doesn't fall into that case, and it doesn't look so bad here. OTOH, what's your suggestion for renaming it?

Reply:
Something like naming the right object 

EncompassingEncounterIds.nameOfSpecificId

Reply:
made it into this: `EncompassingEncounterIds.newId`